### PR TITLE
Propagate consumer group status in Trigger and fallback to Broker delivery spec

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

/kind cleanup

### CODE REVIEW

File: control-plane/pkg/reconciler/trigger/v2/triggerv2.go [hunk 1]
```diff
@@ -187,6 +175,32 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, trigger *eventin
...
+	return broker.Spec.Delivery
+}
+
+func propagateConsumerGroupStatus(cg *internalscg.ConsumerGroup, trigger *eventing.Trigger) {
+	if cg.IsReady() {
+		trigger.Status.MarkDependencySucceeded()
+	} else {
+		topLevelCondition := cg.GetConditionSet().Manage(cg.GetStatus()).GetTopLevelCondition()
+		if topLevelCondition == nil {
+			trigger.Status.MarkDependencyUnknown("failed to reconcile consumer group", "consumer group not ready")
```
Comment (aavarghese) [inline control-plane/pkg/reconciler/trigger/v2/triggerv2.go:192]: change to `"consumer group is not ready"` similar to KafkaSource?

Comment (pierDipi) [reply control-plane/pkg/reconciler/trigger/v2/triggerv2.go]: Done

**Review by aavarghese — APPROVED** [review]
/lgtm

**Pull request merged**